src/pages/Menu.js

```diff
@@ -1,9 +1,11 @@
 import React from "react";
+import {Menus} from '../components';
 
 function Menu(props) {
   return (
     <div class="container">
-      <h1>Menu</h1>
+      <h1 className="py-5">Menu</h1>
```

Comment:
Did you mean for this title to still be included? Asking as it's not in the wireframe.

Reply:
Thanks for the feedback - the H1is  not in the wireframe but happy for it to stay in if you are.

Reply:
yes sure